### PR TITLE
populateRelations function for populating the list of relations in the relations page

### PR DESCRIPTION
**Pull request by puffinpoof**

### CODE REVIEW

File: screens/Relations/Relations.js [hunk 1]
```diff
@@ -31,30 +31,53 @@ class RelationsComponent extends Component {
...
     }
 
+
+
+    populateRelations = () => {
+        return _.map(this.props.all_relations, (item, index)=>{
+            const toDetails = NavigationActions.navigate({
+                routeName: 'Details',
+              
+                params: {},
```
Comment (Mark-Jung) [inline screens/Relations/Relations.js]: We need to add a param here, probably the item's id so that we can populate the details screen.

**Review by Mark-Jung — CHANGES_REQUESTED** [review]
Aside from a minor fix, lgtm

File: screens/Relations/Relations.js (hunk 1 above)
Comment (Mark-Jung) [reply screens/Relations/Relations.js]: Follow the article
https://medium.com/@shovonroy/how-to-properly-navigate-with-react-navigation-e38fe3bf7381
and it should be an easy fix.

**Review by Mark-Jung — CHANGES_REQUESTED** [review]

**Comment by FOBkid20** [comment]
taken care of in other PRs

**Pull request closed without merging**